xkcd_get.py

```diff
@@ -0,0 +1,48 @@
+import requests
+import logging
+import os
+from PIL import Image
+from utility import is_stale, configure_logging
+import sys
+
+configure_logging()
+
+def xkcd_get_img():
+    xkcd_file_name = "xkcd-comic-strip.png"
+    if not is_stale(xkcd_file_name, 3600):
+        logging.info("xkcd-comic-strip.png is still fresh. Skipping download.")
+        sys.exit(1)
```

Comment:
Deliberately exiting with a non-zero exit code, so that the run.sh script doesn't try to refresh the display, just ignores it.